File: Everything/Models/ToDoList/ToDoItemTask.cs
namespace everything.Models
{
    public class ToDoItemTask
    {
        public int Id { get; set; }
        public int Sequence { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool Completed { get; set; }

        public int ToDoItemId { get; set; }
        public virtual ToDoItem ToDoItem { get; set; }
    }
}

File: Everything/ClientApp/src/models/todo/ToDoItemTask.ts
export default class ToDoItemTask {
    id: number = 0;
    toDoItemId: number = 0;
    name: string | null = null;
    description: string = '';
    sequence: number = 0;

    constructor(init?: Partial<ToDoItemTask>) {
        Object.assign(this, init);
    }
}